src/views/document/RouteView.vue

```diff
@@ -77,7 +77,13 @@
 
         <div class="box">
           <markdown-section :document="document" :field="fields.summary" />
-          <markdown-section :document="document" :field="fields.route_history" />
+          <markdown-section v-if="locale.route_history" :document="document" :field="fields.route_history" />
+            <div v-else-if="activities.includes('snow_ice_mixed') || activities.includes('mountain_climbing') || activities.includes('rock_climbing') || activities.includes('ice_climbing') || activities.includes('via_ferrata') || activities.includes('slacklining') || activities.includes('skitouring') && ['5.1', '5.2', '5.3', '5.4', '5.5','5.6'].includes(doc.ski_rating))" class="missing-history-banner no-print"
```

Comment:
If there is no styling applied, I'm pretty sure it will look ugly :)

Reply:
@AntoineMarnat , just run `npm run lint` before committing, it'll  clean your code.

Reply:
What kind of IDE are you using @AntoineMarnat? If you use VS Code (which I would recommend), you can install the prettier plugin and configure it to format on save. This would ease your burden.

Reply:
I tried to use Github desktop and now VS Code, but somehow I fail to push commits ... so I edit directly on github website :/